device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -19,6 +19,14 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin {
         let allDay: Bool
         let attendees: [Attendee]
         let location: String?
+        let recurrenceRule: RecurrenceRule?
+    }
+    
+    struct RecurrenceRule: Codable {
+        let recurrenceFrequency: Int64
```

Comment:
Might be missing something but is there a reason why the `recurrenceFrequency` is of type `Int64` whereas other types are `Int`?

Reply:
actually, shouldn't be needed since there's it only has a limited number of values and will be changed to an enum, will change to Int